From b657edaa59378b4f345fec916059213336d6bc05 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Wed, 26 Feb 2020 18:36:20 +0800
Subject: [PATCH] Rename thumbnail_grid to avoid conflict with new
 kdeplasma-addons
---
 configures/kwinrc                                         | 2 +-
 tabbox/CMakeLists.txt                                     | 2 +-
 tabbox/{thumbnail_grid => chameleon}
/contents/ui/main.qml | 0
 tabbox/{thumbnail_grid => chameleon}
/metadata.desktop     | 6 +++---
 4 files changed, 5 insertions(+), 5 deletions(-)
 rename tabbox/{thumbnail_grid => chameleon}
/contents/ui/main.qml (100%)
 rename tabbox/{thumbnail_grid => chameleon}
/metadata.desktop (71%)
diff --git a/configures/kwinrc b/configures/kwinrc
index 3134b0a78..66a771d8a 100644
--- a/configures/kwinrc
+++ b/configures/kwinrc
@@ -40,7 +40,7 @@ ActivitiesMode=1
 ApplicationsMode=0
 DesktopMode=1
 HighlightWindows=true
-LayoutName=thumbnail_grid
+LayoutName=chameleon
 MinimizedMode=0
 MultiScreenMode=0
 ShowDesktopMode=0
diff --git a/tabbox/CMakeLists.txt b/tabbox/CMakeLists.txt
index af1c0d929..d377f3056 100644
--- a/tabbox/CMakeLists.txt
+++ b/tabbox/CMakeLists.txt
@@ -1,6 +1,6 @@
 install(
     DIRECTORY
-    thumbnail_grid
+    chameleon
     DESTINATION
     ${CMAKE_INSTALL_PREFIX}
/share/kwin/tabbox
 )
diff --git a/tabbox/thumbnail_grid/contents/ui/main.qml b/tabbox/chameleon/contents/ui/main.qml
similarity index 100%
rename from tabbox/thumbnail_grid/contents/ui/main.qml
rename to tabbox/chameleon/contents/ui/main.qml
diff --git a/tabbox/thumbnail_grid/metadata.desktop b/tabbox/chameleon/metadata.desktop
similarity index 71%
rename from tabbox/thumbnail_grid/metadata.desktop
rename to tabbox/chameleon/metadata.desktop
index 7114afe0b..97ae44aa3 100755
--- a/tabbox/thumbnail_grid/metadata.desktop
+++ b/tabbox/chameleon/metadata.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Name=Thumbnail Grid
+Name=Chameleon
 Comment=

 Type=Service
@@ -11,7 +11,7 @@ X-Plasma-MainScript=ui/main.qml
 Icon=preferences-system-windows-switcher-big-icons
 X-KDE-PluginInfo-Author=Chris Holland
 X-KDE-PluginInfo-Email=zrenfire@gmail.com
-X-KDE-PluginInfo-Name=thumbnail_grid
+X-KDE-PluginInfo-Name=chameleon
 X-KDE-PluginInfo-Version=4
-X-KDE-PluginInfo-Website=https://github.com/Zren/kwin-tabbox-thumbnail_grid
+X-KDE-PluginInfo-Website=https://github.com/linuxdeepin/dde-kwin
 X-KDE-PluginInfo-Depends=
